grassroots-shared/tsconfig.json

```diff
@@ -3,5 +3,5 @@
   "compilerOptions": {
     "outDir": "./dist"
   },
-  "include": ["src"]
+  "include": ["src", "../grassroots-backend/src/testing/CreateDTOBase.spec.ts"]
```

Comment:
are there best practices regarding crossing the package boundary and when it's OK?

Reply:
I have no idea why I was crossing the package boundary here. This seems completely unnecessary...

Removed it, and I'll see if anything explodes.

In general, I think the best practice is to never cross the package boundary, but in practice it sometimes makes things significantly simpler. (But not here, as far as I can tell)